Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
I understand.

Gotta admit that while reading the code, some things could have been done differently: `support_tools.info_collector.registry` could be named `support_tools.system_info.registry_collector`, or `support_tools.system_info.collector.registry` (e.g. always prefix "info" with "collector", separated by `_` to indicate that it is one word ("info_collector" not being a name in the domain of ez support tools).